tox.ini

```diff
@@ -38,7 +38,16 @@ source =
 [flake8]
 max-line-length = 90
 extend-ignore = S101, D101, D102, D103, D105, D106, D107, TC006
-per-file-ignores = flake8_trio/visitors/__init__.py: F401, E402
+extend-enable = TC10
```

Comment:
https://github.com/snok/flake8-type-checking#choosing-how-to-handle-forward-references